cloudant-sync-datastore-core/src/main/java/com/cloudant/mazha/CouchClient.java

```diff
@@ -419,11 +419,47 @@ public void putAttachmentStream(String id, String rev, String attachmentName, St
         });
     }
 
-    public Iterable<DocumentRevsList> bulkReadDocsWithOpenRevisions(List<BulkGetRequest> request) {
-        // not implemented yet
-        // the idea here is to provide an iterator which will pull more data from the
-        // response stream each time next() is called
-        return null;
+    public Iterable<DocumentRevsList> bulkReadDocsWithOpenRevisions(List<BulkGetRequest> request, boolean pullAttachmentsInline) {
```

Comment:
Should this method be so replication specific? E.g., hard-coding `revs=true` and the fact the parameter is calls `pullAttachmentsInline` rather than `getAttachmentsInline` or just `inlineAttachments` (pull is a replication term).